### ISSUE TITLE
This plugin is only for docz v1 and breaks docz v2

### ISSUE THREAD

**Issue by trybick** [pedronauck/docz-plugin-css#32]
If this is true, the docs on this repo should be updated to say **this only works with docz v1**.

For anyone wondering, here is the way to do css preprocessors in docz v2
https://www.docz.site/docs/usage-with-css-preprocessors

**Comment by theKashey** [comment]
Yep. Adding `docz-plugin-css` is breaking `docz` in a hard to understand way - the `docz-core` at the top level would be `0.11`, while your `docz` is `2.2`.

You will not understand the problem unless check versions in `yarn.lock`
